ONSdigital/eq-questionnaire-runner
# Exclude test schemas from make clean

### What is the context of this PR?
Updates the `make clean` command so that it doesn't delete the test schemas but deletes all others
Also a test schema that was added in a recent PR has been moved to the new schemas/test directory

### How to review 
- Run `make load-schemas`
- Run `make clean`
- See that all folders in the schemas directory have been deleted except the "test" directory

### Checklist
